usaspending_api/references/urls_v2.py

```diff
@@ -11,5 +11,6 @@
 
 
 urlpatterns = [
-    url(r'^agency/(?P<pk>[0-9]+)/$', agency.AgencyViewSet.as_view())
+    url(r'^agency/(?P<pk>[0-9]+)/$', agency.AgencyViewSet.as_view()),
+    url(r'^toptier_agency/$', toptier_agency.ToptierAgencyViewSet.as_view())
```

Comment:
If we're returning a list, is it better to pluralize the endpoint to indicate many results?